tests/unittests/tests-core/tests-core-clist.c

```diff
@@ -237,13 +237,19 @@ static int _foreach_test_trampoline(clist_node_t *node, void *arg)
 
 static void test_clist_foreach(void)
 {
+    void *res;
```

Comment:
why not moving the declaration of `res` line 24 ?

Reply:
It is used three times after to do `res = clist_foreach(...)`, so it makes these lines the same.